tests/common.sh

```diff
@@ -38,7 +38,7 @@ wait_version() {
     output=$(../bin/aergocli blockchain 2>/dev/null)
     # check if 'output' is non-empty and starts with '{'
     if [[ -n "$output" ]] && [[ "${output:0:1}" == "{" ]]; then
-      cur_version=$(echo "$output" | jq .ChainInfo.Chainid.Version | sed 's/"//g')
+      cur_version=$(echo "$output" | jq .chainInfo.id.version | sed 's/"//g')
```

Comment:
Why did you change this?
I was expecting backwards compatibility
BTW, it is failing on integration tests and this change can be one of the causes

Reply:
The changes MUST be backwards compatible, otherwise the existing systems will stop working!

Reply:
it is change about aergocli response type. ( we change all json key format to camelCase )
can this be a compatibility issue? i thought cli is just viewer, like json api.

Reply:
There are many integration tests on other PRs that are waiting to be merged on V4
This change will make all them stop working
Not good

Reply:
Unless you fix them all later

Reply:
sorry about that, but this change is necessary as all keys need to follow the same format.
after merge, I will fix all integration tests in v4 hardfork branch.

Reply:
it is not on a single branch, there are many of them
each PR is on a separate branch